docs/TranslationLocales.md

```diff
@@ -23,7 +23,7 @@ You can find translation packages for the following languages:
 - Farsi (`fa`): [hamidfzm/ra-language-farsi](https://github.com/hamidfzm/ra-language-farsi)
 - Finnish (`fi`): [aikain/ra-language-finnish](https://github.com/aikain/ra-language-finnish)
 - French (`fr`): [marmelab/ra-language-french](https://github.com/marmelab/react-admin/tree/master/packages/ra-language-french)
-- German (`de`): [greenbananaCH/ra-language-german](https://github.com/greenbananaCH/ra-language-german) (tree translation: [straurob/ra-tree-language-german](https://github.com/straurob/ra-tree-language-german))
```

Comment:
Its not only `(translation)` but `(formal and informal translation packages available)`, the `(translation)` is only what git keeps from the previous version.
The package exports both `formalGermanMessages` as well as `informalGermanMessages`, since German uses different words for those forms, hence the explanation in the brackets. But feel free to remove it it you don't think it's needed.